src/style.css

```diff
@@ -2,2 +1,69 @@
+body,
+html {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  background-color: #858585;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
You are adding a right margin to all `.nav__item` elements except the last one. This is a good practice as it ensures that the items are evenly spaced and there is no extra space after the last item.